docs/core/porting/project-structure.md

```diff
@@ -0,0 +1,79 @@
+# Organizing Your Project to Support .NET Framework and .NET Core
+
+This article is to help project owners who want to compile their solution against .NET Framework and .NET Code side-by-side.  It provides several options to organize projects to help developers achieve this goal.
+
+## General Guidance
+
+1.  Create an xproj that targets multiple frameworks.
```

Comment:
Do all customers know what xproj is? Maybe give its proper name here and put .xproj in parentheses?